axelor-human-resource/src/main/resources/views/TimesheetLine.xml

```diff
@@ -310,7 +310,7 @@
   		<context name="_project" expr="eval: project"/>
   		<context name="_user" expr="eval: assignedTo"/>
   		<context name="_company" expr="eval: project.company"/>
-  		<context name="_teamTaskInvoicing" expr="eval: teamTaskInvoicing"/>
+  		<context name="_teamTaskInvoicing" expr="eval: toInvoice"/>
```

Comment:
also update attribute in action-team-task-attrs-default-timesheet-line accordingly